packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -119,6 +119,9 @@ async function createIncomingPayment(
   if (!paymentPointer) {
     return IncomingPaymentError.UnknownPaymentPointer
   }
+  if (!paymentPointer.isActive) {
+    return IncomingPaymentError.InactivePaymentPointer
+  }
```

Comment:
Do I need to do this same check in the `incoming_remote` create? I'm not quite clear on what a remote incoming payment is. Does that use an external payment pointer by chance (and therefore doesnt need this check)?